api/src/main/java/io/minio/MinioClient.java

```diff
@@ -3158,58 +3209,58 @@ public void putObject(String bucketName, String objectName, InputStream stream,
    *
    * @param bucketName
    *          Bucket name.
-   * 
+   *
    * @param objectName
    *          Object name to create in the bucket.
-   * 
+   *
    * @param stream
    *          stream to upload.
-   * 
+   *
    * @param size
    *          Size of all the data that will be uploaded.
-   * 
+   *
    * @param contentType
    *          Content type of the stream.
-   * 
+   *
    * @param key
    *          Master key for encryption.
    *
    * @throws InvalidBucketNameException
    *           upon invalid bucket name is given
-   * 
+   *
    * @throws NoResponseException
    *           upon no response from server
-   * 
+   *
    * @throws IOException
    *           upon connection error
-   * 
+   *
    * @throws XmlPullParserException
    *           upon parsing response xml
-   * 
+   *
    * @throws ErrorResponseException
    *           upon unsuccessful execution
-   * 
+   *
    * @throws InternalException
    *           upon internal library error
-   * 
+   *
    * @throws InvalidAlgorithmParameterException
    *           upon wrong encryption algorithm used
-   * 
+   *
    * @throws BadPaddingException
    *           upon incorrect padding in a block
-   * 
+   *
    * @throws IllegalBlockSizeException
    *           upon incorrect block
-   * 
+   *
    * @throws NoSuchPaddingException
    *           upon wrong padding type specified
    *
    * @see #putObject(String bucketName, String objectName, String fileName, String contentType)
-   * 
+   *
    * @see #putObject(String bucketName, String objectName, String fileName)
-   * 
+   *
    * @see #putObject(String bucketName, String objectName, InputStream stream, long size, String contentType)
-   * 
+   *
```

Comment:
Unwanted line modification makes review harder.  Its always recommended to send a separate PR for such cleanup than mingling with new API addition.